nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -92,8 +93,8 @@ contract StakingEscrow is Issuer {
         uint16 lastActivePeriod;
         bool measureWork;
         uint256 completedWork;
+        bool windDown;
```

Comment:
I thought that `windDown` was going to be `true` by default? Consequently, the field would be something like `windDownDisabled`.

Reply:
Oh I see in your PR description that the default is False. If this was the consensus, then there's no comment here :)

Reply:
Yep, it was consensus, False by default